ptp/utils_test.go

```diff
@@ -0,0 +1,48 @@
+package ptp
+
+import (
+	"math"
+	"testing"
+)
+
+func TestNormalizeAngle(t *testing.T) {
```

Comment:
Test functions should use Go conventions:

```suggestion
func Test_normalizeAngle(t *testing.T) {
```